FF1Lib/TreasureStacks.cs

```diff
@@ -0,0 +1,65 @@
+﻿using RomUtilities;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FF1Lib
+{
+	public enum TreasureStackSize
+	{
+		[Description("Default")]
+		Default,
+
+		[Description("Stingy (2 Tents, 5 Heals)")]
```

Comment:
Alright this looks good, just put the quantities in the tooltip and I'll merge this tomorrow.

Reply:
ok